### ISSUE TITLE
forward_client_headers doesn't not forward accept-language header to actions

### ISSUE THREAD

**Issue by Zerebokep** [hasura/graphql-engine#9211]
Please forgive me not using the template, but everything is basically described in the title.

The accept-language request header is not forwarded when forward_client_headers is set, I don't think this is intended as `accept-encoding` is forwarded too.

**Comment by adas98012** [comment]
@Zerebokep 
We need more context about your use case. For example, are you using a webhook and if so is it a GET or POST request ?
Please provide the additional info (redacted as applicable).

Here is the relevant [Documentation](https://hasura.io/docs/latest/auth/authentication/webhook/#get-request) (about headers forwarded in an authentication webhook), for your reference.

**Comment by Zerebokep** [comment]
Yes, I'm using a webhook with the "Forward client headers to webhook" checkbox ticked, but not all of the mentioned headers from the docs are forwarded.

I want to save the users browser language in my database for later use.

**Comment by adas98012** [comment]
@Zerebokep 
Thanks for the additional info. Just curious what are you using in your application:  Hasura Cloud or OSS ?

**Comment by Zerebokep** [comment]
I currently use the hasura/graphql-engine:latest.cli-migrations-v3 docker image.

**Comment by Rodeoclash** [comment]
I'd just like to say that I also ran into this problem and I'm unsure exactly what the "Forward client headers to webhook" tickbox even does.

My use case is that I run Ory Oathkeeper to automatically take a session and convert that to headers as requests flow through the proxy. The end result of this is that I get an `X-User-Id` header. To save additional lookups about a user, I'd also like to pass other fields in the header like `X-User-Email`, however, these headers never make it to the action endpoint.

I've tested this with both `GET` and `POST` requests and neither seem to contain the additional headers despite what I name them (i.e. avoiding the `X-User` prefix).

My expectation is that clicking that forward headers would either merge the headers that Hasura uses with those from the client somehow to get a payload containing both, or in the event of a `POST`, that it would contain an extra field in the body which contained the original headers sent by the client.

I'm seeing this behaviour in the self hosted version of Hasure, version 2.22.0 (using Docker)
